### DISCUSSION TITLE
My-Linux-Config/docs/ccls-vs-clangd

### DISCUSSION THREAD

**Discussion by giscus** [Martins3/My-Linux-config#163] [General]
# My-Linux-Config/docs/ccls-vs-clangd

:clap: Modern neovim configuration based on native lsp

https://martins3.github.io/My-Linux-Config/docs/ccls-vs-clangd.html

**Comment by ad56583964** [comment]
2025 年了...好像反转了...

以 Linux Kernel 5.10 为例
include/linux/sunrpc/xdr.h 中
 __be32 识别不出...
伴随着的 xdr_decode_opaque_fixed 也识别不出，而 clangd 可以成功跳转...

**Reply by Martins3** [reply]
有趣，我可以重新尝试下
